capa/render/proto/__init__.py

```diff
@@ -136,7 +137,8 @@ def metadata_to_pb2(meta: rd.Metadata) -> capa_pb2.Metadata:
             arch=meta.analysis.arch,
             os=meta.analysis.os,
             extractor=meta.analysis.extractor,
-            rules=meta.analysis.rules,
+            # TODO convert analysis.rule type to Path in capa_pb2.Metadata
```

Comment:
capa_pb2.py is generated by protobuf so we aren't able to update there. if necessary, Metadata can contain a str instead of Path, due to the serialization requirements of protobuf.